### ISSUE TITLE
Netherlands tests not updated in a while.

### ISSUE THREAD

**Issue by ppetermann** [owid/covid-19-data#171]
The amount of tests in the netherlands hasn't been updated in about a month. 

the following website seems to have official numbers for the previous two weeks - its the best i could find with my lack of dutch, but maybe it would help to parse that page, to fill in that data? 

https://www.rivm.nl/en/novel-coronavirus-covid-19/current-information

**Comment by edomt** [comment]
Hi @ppetermann 
As noted in [our documentation](https://ourworldindata.org/coronavirus-testing#netherlands), there appear to be large reporting lags in Dutch testing data, such that the number of labs reporting for days in recent weeks tend to be lower. As a result, the reported figures are updated retrospectively as additional labs report their results. The figures we display are based on the latest available epidemiological report, but we currently censor the last month of data to avoid showing an artificially low number of tests (and therefore an overestimated positive rate).

**Comment by jwulffr** [comment]
Hi @edomt, The PDF file you reference does indeed get updated for earlier weeks. 

Would it not be possible for you to fully (re)load the table from the file and overwrite your existing information? The table in the pdf includes the information for all the weeks so far, both the performed tests as well as the positive tests. 

The changes to the historical data have a limited impact on the positive rate ( <1% change ). 
It would really be helpful to have the information included even though it is not yet 100% accurate.

As dutch is my native language I would be more than willing to clarify any questions you might have on the pdf,

**Comment by edomt** [comment]
If we look at the report published on [October 6](https://www.rivm.nl/sites/default/files/2020-10/COVID-19_WebSite_rapport_wekelijks_20201006_1159_metRt.pdf), it shows 186777 people tested for the week of 2020-09-28 - 2020-10-04. For that same week, the latest report published on [November 3](https://www.rivm.nl/sites/default/files/2020-11/COVID-19_WebSite_rapport_wekelijks_20201103_1216.pdf) shows a total of 221677 people tested. That's a 19% increase in the number of people tested, which means that a positive rate calculated based on the first report would have overestimated the positive rate by a factor of 1.2. (And of course there's no guarantee that this Nov 3 report was the final number for that week.)

At a given time, the drop in the number of tests happens around 30 days prior. This is what the data looks like based on the November 3 report:
![image](https://user-images.githubusercontent.com/7103774/98683230-166f6400-2365-11eb-891a-36d527e39190.png)

So we know (based on what happened for previous reports) that this fall in the number of tests is an artifact of the data collection process, and that the numbers for the same period will most likely increase in the following reports. And in the meantime, if we publish the numbers as they are, the positive rate will be extremely high as a consequence: 
![image](https://user-images.githubusercontent.com/7103774/98683483-60f0e080-2365-11eb-9463-3b799ca1862e.png)

Of course the positive rate in the Netherlands _is_ probably high at the moment, but we'd rather introduce a delay in the availability of the data than show an overestimated positive rate.

As a final note, it seems that this issue has worsened recently: for the week of 2020-08-31 – 2020-09-06, the report published 9 days later was already including 99.6% of the final testing tally for that week. As in many other EU countries, I'm guessing that the high number of cases and the labs being overwhelmed with tests has made reporting delays much worse in the current period.

**Comment by jwulffr** [comment]
Dear @edomt,

Thank you for the feedback. Based on your numbers you are using the table in chapter 11?
Please be aware that numbers in this table should only be be used in the context of this table & are not 100% relatable to reported cases. Not all positive COVID19 cases reported are included in this table! (& the numbers for the last line/period are never complete)

For example the period 2020-09-28 - 2020-10-04 in your data gives 23228 cases:
![image](https://user-images.githubusercontent.com/62597703/98700926-06fa1600-2379-11eb-96df-fa58b6d23938.png)

While the table in chapter "only" gives 20027 positive people tested for this period. 

The table in chapter 11 depicts the number of people tested ("Aantal geteste personen") vs the number of people tested positive in that group ("Aantal positief") + positivity rate based on these 2 numbers ("% positief")

![image](https://user-images.githubusercontent.com/62597703/98696861-90f3b000-2374-11eb-8a6c-a35e63b556ec.png)

While the number of  people tested ("Aantal geteste personen") and the number of people tested positive in that group ("Aantal positief") included in this table will vary between releases of this report the resulting positivity rate based on these 2 numbers ("% positief") only changes <1% (0.3% for the period 2020-09-28 - 2020-10-04)

In short, you can use the rightmost column to show a close estimation of the positivity rate (or verify your own estimations) but the values should not be  (directly) used in calculations vs the case count you also have.

**Comment by edomt** [comment]
Yes, this is what we realized only recently: that the cases reported in this table were much lower than the cases that we have in our data (that we take from the European CDC). Does that mean that more people are being tested than what is shown in this table? Is this data available somewhere?

**Comment by EwoutH** [comment]
On https://data.rivm.nl/covid-19/ there is a daily updated file available called [COVID-19_uitgevoerde_testen.csv](https://data.rivm.nl/covid-19/COVID-19_uitgevoerde_testen.csv) (also in [json](https://data.rivm.nl/covid-19/COVID-19_uitgevoerde_testen.json)) which lists the number of tests performed and number of tests positive for each Dutch [Safety region](https://en.wikipedia.org/wiki/Safety_region). It updated daily at 15:15 and contains data up to two days prior (so today, November 19th, it contains the data up to November 17th).

It's also the exact data used on the national [coronadashboard](https://coronadashboard.government.nl/landelijk/positief-geteste-mensen) for The Netherlands.
![covid-testing-rate-nl](https://user-images.githubusercontent.com/15776622/142642308-98db1ffa-51a8-48a7-a38e-07195ad72115.png)

So currently the most up to data on Our World in Data is from November 7th, a lag of 12 days. So this lags could be reduced to only 2 days.

**Comment by camappel** [comment]
Hello Ewout,

Thank you very much for your email. You are correct, currently our testing
data for the Netherlands has a lag of up to 12 days due to our update
schedule (weekly data for the previous week is published late on Tuesday
and we perform weekly updates on Tuesdays and Fridays.

We were not previously aware of the daily data so thank you very much for
bringing this to our attention. We will perform due diligence on this
source; if it meets our criteria
<https://ourworldindata.org/coronavirus-testing#our-checklist-for-covid-19-testing-data>,
it will allow us to significantly reduce the lag in Netherlands testing
data, as you say.

Thanks again for your contribution!
Cameron Appel

**Comment by EwoutH** [comment]
I created a new issue with more details, including a nice R script that reads the daily data published above.
- #2149
